Nightly GCR image cleanup workflow fails

### Description

The nightly GCR image cleanup workflow failed with the following log:

```
2023-05-31T00:05:47.0285441Z + for digest in $***ALL_DIGESTS[@]***
2023-05-31T00:05:47.0286862Z + gcloud container images delete --force-delete-tags -q gcr.io/mirrornode/hedera-mirror-rest-monitor@sha256:6209c298a1e16eee72cb9ee65ac06c9fefd4be2355e5d4531679cc4aa86e6587
2023-05-31T00:05:47.8222112Z Digests:
2023-05-31T00:05:47.8223851Z - gcr.io/mirrornode/hedera-mirror-rest-monitor@sha256:6209c298a1e16eee72cb9ee65ac06c9fefd4be2355e5d4531679cc4aa86e6587
2023-05-31T00:05:48.8110033Z Deleted [gcr.io/mirrornode/hedera-mirror-rest-monitor@sha256:6209c298a1e16eee72cb9ee65ac06c9fefd4be2355e5d4531679cc4aa86e6587].
2023-05-31T00:05:48.9363765Z + for digest in $***ALL_DIGESTS[@]***
2023-05-31T00:05:48.9365864Z + gcloud container images delete --force-delete-tags -q gcr.io/mirrornode/hedera-mirror-rest-monitor@sha256:f0d53c60015d36de10b045abcdef9f334ebfb58a884eb3bb404262e45ce385b4
2023-05-31T00:05:49.7905012Z Digests:
2023-05-31T00:05:49.7906608Z - gcr.io/mirrornode/hedera-mirror-rest-monitor@sha256:f0d53c60015d36de10b045abcdef9f334ebfb58a884eb3bb404262e45ce385b4
2023-05-31T00:05:50.8678453Z Deleted [gcr.io/mirrornode/hedera-mirror-rest-monitor@sha256:f0d53c60015d36de10b045abcdef9f334ebfb58a884eb3bb404262e45ce385b4].
2023-05-31T00:05:51.0007743Z + for digest in $***ALL_DIGESTS[@]***
2023-05-31T00:05:51.0009347Z + gcloud container images delete --force-delete-tags -q gcr.io/mirrornode/hedera-mirror-rest-monitor@sha256:4ae5f97e137c0c8df681e0a54ccc112a89d89ebc0cf2377c79b89b922a5c3798
2023-05-31T00:05:51.8307615Z Digests:
2023-05-31T00:05:51.8309188Z - gcr.io/mirrornode/hedera-mirror-rest-monitor@sha256:4ae5f97e137c0c8df681e0a54ccc112a89d89ebc0cf2377c79b89b922a5c3798
2023-05-31T00:05:52.8983356Z Deleted [gcr.io/mirrornode/hedera-mirror-rest-monitor@sha256:4ae5f97e137c0c8df681e0a54ccc112a89d89ebc0cf2377c79b89b922a5c3798].
2023-05-31T00:05:53.0247862Z + for digest in $***ALL_DIGESTS[@]***
2023-05-31T00:05:53.0249522Z + gcloud container images delete --force-delete-tags -q gcr.io/mirrornode/hedera-mirror-rest-monitor@sha256:246259cb7efb924f913583833b44abe81819cca6aa65d836a745e2a61a4c7f21
2023-05-31T00:05:53.7958380Z Digests:
2023-05-31T00:05:53.7959995Z - gcr.io/mirrornode/hedera-mirror-rest-monitor@sha256:246259cb7efb924f913583833b44abe81819cca6aa65d836a745e2a61a4c7f21
2023-05-31T00:05:54.8627530Z Deleted [gcr.io/mirrornode/hedera-mirror-rest-monitor@sha256:246259cb7efb924f913583833b44abe81819cca6aa65d836a745e2a61a4c7f21].
2023-05-31T00:05:54.9885187Z + for digest in $***ALL_DIGESTS[@]***
2023-05-31T00:05:54.9886727Z + gcloud container images delete --force-delete-tags -q gcr.io/mirrornode/hedera-mirror-rest-monitor@sha256:6209c298a1e16eee72cb9ee65ac06c9fefd4be2355e5d4531679cc4aa86e6587
2023-05-31T00:05:55.7707322Z Digests:
2023-05-31T00:05:55.7711869Z - gcr.io/mirrornode/hedera-mirror-rest-monitor@sha256:6209c298a1e16eee72cb9ee65ac06c9fefd4be2355e5d4531679cc4aa86e6587
2023-05-31T00:05:56.1774829Z ERROR: (gcloud.container.images.delete) Not found: response: ***'docker-distribution-api-version': 'registry/2.0', 'content-type': 'application/json', 'content-encoding': 'gzip', 'date': 'Wed, 31 May 2023 00:05:56 GMT', 'server': 'Docker Registry', 'cache-control': 'private', 'x-xss-protection': '0', 'x-frame-options': 'SAMEORIGIN', 'alt-svc': 'h3=":443"; ma=2592000,h3-29=":443"; ma=2592000', 'transfer-encoding': 'chunked', 'status': 404***
2023-05-31T00:05:56.1778561Z Failed to compute blob liveness for manifest: 'sha256:6209c298a1e16eee72cb9ee65ac06c9fefd4be2355e5d4531679cc4aa86e6587': None
2023-05-31T00:05:56.3028100Z ##[error]Process completed with exit code 1.
```

It appears the image hash `6209c298a1e16eee72cb9ee65ac06c9fefd4be2355e5d4531679cc4aa86e6587` appeared twice in the `ALL_DIGESTS` array, thus the script failed to delete the same image the second time.

### Steps to reproduce

Check the logs of failed workflow run

### Additional context

_No response_

### Hedera network

other

### Version

v0.81.0

### Operating system

None

--- a/.github/workflows/cleanup.yml
+++ b/.github/workflows/cleanup.yml
@@ -53,6 +53,9 @@ jobs:
             ALL_DIGESTS+=(${CHILD_DIGESTS[@]})
           done
           
+          # dedup the digests since images may be shared by list type images
+          ALL_DIGESTS=($(printf '%s\n' "${ALL_DIGESTS[@]}" | sort -u))
+          
           for digest in ${ALL_DIGESTS[@]}; do
             gcloud container images delete --force-delete-tags -q "${IMAGE_PATH}@${digest}"
           done

